docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -1100,6 +1100,26 @@ dnsSansLocalCertificate()
 Returns the DNS entries (as a table) in the SAN field of the local certificate. Returns an empty
 table if there is no local certificate, or no SAN field, or no DNS SAN entries.
 
+oidsPeerCertificate()
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+
+  downstreamSslConnection:oidsPeerCertificate()
```

Comment:
Please clearly document that the returned table is an array of OID strings (and not the values).